src/main/java/mekanism/common/item/gear/ItemMekaTool.java

```diff
@@ -233,20 +248,25 @@ public boolean onBlockStartBreak(ItemStack stack, BlockPos pos, Player player) {
             Level world = player.level;
             BlockState state = world.getBlockState(pos);
             boolean silk = isModuleEnabled(stack, MekanismModules.SILK_TOUCH_UNIT);
-            FloatingLong energyRequired = getDestroyEnergy(stack, state.getDestroySpeed(world, pos), silk);
+            float targetHardness = state.getDestroySpeed(world, pos);
+            FloatingLong energyRequired = getDestroyEnergy(stack, targetHardness, silk);
```

Comment:
This change is technically unneeded as the targetHardness isn't used here outside where it was before. EXCEPT one thing I realized is in terms of energy cost for when blasting is occurring this should use the hardness of the hardest block in the area instead of just the one being directly mined.

Reply:
Yup, I was passing it to the blasting before changing to the interface. About the energy though, when I looked the only effect hardness had on it was that if it was 0 (which we aren't blasting) then the energy was halved. Also I think that should still be the target block energy as it will still be mined first when blasting and the other blocks will follow the same way as vein mined ones, cumulatively adding energy required.

Reply:
Ah wait I was forgetting how the energy cost worked, you are right it shouldn't matter and should be the actual target block.